src/goto-programs/property_checker.h

```diff
@@ -1,55 +0,0 @@
-/*******************************************************************\
-
-Module: Property Checker Interface
-
-Author: Daniel Kroening, kroening@kroening.com
-
-\*******************************************************************/
-
-/// \file
-/// Property Checker Interface
-
-#ifndef CPROVER_GOTO_PROGRAMS_PROPERTY_CHECKER_H
-#define CPROVER_GOTO_PROGRAMS_PROPERTY_CHECKER_H
-
-// this is just an interface -- it won't actually do any checking!
-
-#include <util/message.h>
-
-#include "goto_trace.h"
-#include "goto_model.h"
-
-class property_checkert:public messaget
```

Comment:
This is used by Deltacheck and 2LS.

Reply:
What is it good for? Would it be possible to contribute documentation? Why does CBMC/JBMC not use it?

Reply:
Moved to #2212 for separate discussion.